chat/conversations.ts

```diff
@@ -524,6 +535,7 @@ class ChannelConversation
 
   close(): void {
     core.connection.send('LCH', { channel: this.channel.id });
+    clearInterval(this.cacheInterval);
```

Comment:
Was it a design decision or an oversight to not clear the conversation cache here too? (same with close on line 673).

Not a right or wrong answer, but since this is done directly through user input I was wondering if you thought it should keep the cached message or not.

Reply:
Yeah, it was deliberate in this case but it's really a judgement call. My thought is to times where I've accidentally closed a tab while trying to click elsewhere, and ended up losing some work. FWIW, it would already reload the last save from disk, unless we wanted this to also deregister the cache (which is an option!)